src/Stratis.Bitcoin/P2P/PeerConnectorDiscovery.cs

```diff
@@ -123,12 +120,15 @@ public override void OnStartConnect()
                 break;
             }
 
-            this.logger.LogTrace("Peer selected: '{0}'", peer?.EndPoint);
-
-            if (peer != null)
+            //If the peer selector returns nothing, we need to wait a bit,
```

Comment:
Maybe add in to the comment more information about how long we are waiting.

Reply:
Fixed 👍